app/controllers/events/view/scheduler.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  header: {
+    left   : 'today prev,next',
+    center : 'title',
+    right  : 'timelineDay,timelineThreeDays,agendaWeek,month'
+  },
+  views: {
+    timelineThreeDays: {
+      type     : 'timeline',
+      duration : { days: 3 }
+    }
+  },
+  actions: {
+    drop() {
+      console.log('drop');
+    },
+    eventReceive(event) {
```

Comment:
@CosmicCoder96 can I get back to you on this later during the day ? I'll have to try it out locally.